hex/lib/dependabot/hex/update_checker/version_resolver.rb

```diff
@@ -122,6 +123,12 @@ def check_original_requirements_resolvable
 
           true
         rescue SharedHelpers::HelperSubprocessFailed => e
+          # TODO: Catch the warnings as part of the Elixir module. This happens
+          # when elixir throws warnings from the manifest files that end up in
+          # stdout and cause run_helper_subprocess to fail parsing the result as
+          # JSON.
+          return error_result(e) if includes_result?(e)
```

Comment:
Will have a poke at the underlying issue, as this is caused by warnings like: 

```
warning: a term is constructed, but never used
  lib/encoder.ex
```

Don't think this is from our code so might be internal hex or customer manifests

Reply:
This change is consistent with the error result handling in `handle_hex_errors`